atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepBatch.java

```diff
@@ -30,6 +30,10 @@
     List<WriteInfo> writes();
     long lastSweptTimestamp();
 
+    default boolean hasNothingToSweep() {
```

Comment:
nit: You can standardise the code a bit by calling this `isEmpty` and just deferring to `writes().isEmpty()`.  I think this reads slightly nicer when it is used too.